openframe/services/openframe-frontend/CLAUDE.md

```
@@ -65,22 +63,22 @@ NEXT_PUBLIC_SHARED_HOST_URL=https://auth.openframe.ai   # Shared auth host
-NEXT_PUBLIC_ENABLE_DEV_TICKET_OBSERVER=true   # Dev ticket auth mode
-NEXT_PUBLIC_FEATURE_SCRIPT_SCHEDULE=false       # Script scheduling (default: false)
-NEXT_PUBLIC_FEATURE_MONITORING=false            # Monitoring pages (default: false)
+NEXT_PUBLIC_ENABLE_DEV_TICKET_OBSERVER=true   # Dev ticket auth mode (Bearer tokens instead of cookies)
 ```
 
+Feature flags are **not** env vars — they are server-loaded via GraphQL (see Feature Flags below). Native-shell env split is documented in `.env.export.example`.
+
 ## Architecture & Structure
 
 ### Technology Stack
 | Category | Technology | Version |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)